src/rulesets/oas/index.ts

```diff
@@ -1,7 +1,10 @@
 import { DiagnosticSeverity } from '@stoplight/types';
 import { FunctionCollection, RuleCollection, RuleFunction, RuleType } from '../../types';
 
-export const operationPath = "$..paths.*[?( name() !== 'parameters' )]";
+export const operationPath =
+  "$..paths.*[?( name() === 'get' || name === 'put' || name === 'post'" +
+  " || name === 'delete' || name === 'options' || name === 'head'" +
```

Comment:
```suggestion
  " || name() === 'delete' || name() === 'options' || name() === 'head' +
```
Ditto

Reply:
Fixed all this and added tests to confirm. Please add thumbs.